docs/examples/data_access.ipynb

```diff
@@ -4,10 +4,39 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Access _PLINDER_ data for training ML models\n",
+    "# PLINDER MLSB Challenge\n",
     "\n",
-    "The goal of this tutorial is to provide a simple and simple hands-on demo for a new user to access _PLINDER_ dataset in prepparation for training machine learning models.\n",
+    "1. [Background information](#background-information)\n",
+    "2. [Accessing and loading data for training](#load-data) \n",
     "\n",
+    "The goal of this tutorial is to provide background information for the MLSB/PLINDER challenge, as well as a simple hands-on demo for how participants can access and use the plinder dataset.\n",
+    "\n",
+    "Specifically, we will cover:\n",
```

Comment:
I removed this enumeration: In the rendered docs, these headings will appear in the sidebar anyway

Reply:
This is a stale notebook, it seems, we should remove it.